packages/firestore/src/util/async_queue.ts

```diff
@@ -275,6 +278,15 @@ export class AsyncQueue {
     );
     this.delayedOperations.push(delayedOp);
 
+    // Fast-forward delays for timerIds that have been overriden.
+    for (const timerIdToSkip of this.timerIdsToSkip) {
+      if (this.containsDelayedOperation(timerIdToSkip)) {
+        this.runDelayedOperationsEarly(timerIdToSkip)
+          .then(() => {})
+          .catch(() => 'obligatory catch');
+      }
+    }
```

Comment:
Rather than use runDelayedOperationsEarly(), I think you could just call `delayedOp.skipDelay()` which should be portable to Android?

Reply:
I forgot to remove this code snippet, but I've replaced whole `timerIdsToSkip` with `removeTransactionBackoffs` to override the exponential backoff directly. Just pushed again